index.html

```diff
@@ -0,0 +1,392 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800&family=Nunito:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Kmarketer</title>
+  </head>
+  <body>
+    <!-- mobile header -->
+    <div class="mobile-nav">
+      <div class="mobile-nav__top">
+        <div class="site-logo">KMarketer</div>
+        <div class="mobile-nav__icons">
+          <img src="./asserts/search.svg" alt="Search" />
+          <img src="./asserts/cart.svg" alt="cart" />
+          <img src="./asserts/menu.svg" alt="menu" />
+        </div>
+      </div>
+      <div class="mobile-nav__lists">
+        <ul class="mobile-nav__lists--center">
```

Comment:
`mobile-nav__lists--center` is included without it's default Element `mobile-nav__lists.`